public/data/R0land013.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Rolando Rio Garaboa",
+  "type": "personal",
+  "bio": "Developer. Always learning!",
+  "avatar": "https://github.com/R0land013.png",
+  "links": [
+    {
+      "name": "Here is my Github",
+      "url": "https://github.com/R0land013",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/r0landorio",
+      "icon": "twitter"
+    },
+    {
+      "name": "My LinkedIn",
+      "url": "https://www.linkedin.com/in/rolando-juan-rio-garaboa-689428214",
+      "icon": "linkedin"
+    },
+    {
+      "name": "You can find me on Dev Community",
+      "url": "https://dev.to/r0land013",
+      "icon": "dev.to"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Computer Engineering studies",
+      "date": "September 2018",
+      "icon": "graduation-hat",
+      "color": "green",
+      "description": "Started studies",
+      "url": ""
```

Comment:
Not sure if this is needed :)

Reply:
Hello. You are right. There is redundant information. I thought all attributes were obligatory 😆 . Thank you.